Sources/AmplitudeiOSSessionReplayMiddleware/AmplitudeiOSSessionReplayMiddleware.swift

```diff
@@ -85,4 +85,8 @@ import AmplitudeSessionReplay
             sessionReplay?.flush()
         }
     }
+
+    public func amplitudeDidRemove(_ amplitude: Amplitude) {
```

Comment:
Question:
Should this be `amplitudeDidRemoveMiddleware` based on [this](https://github.com/amplitude/Amplitude-iOS/blob/ccb9e4fdf2871308c4085e42e8b4e12c9906cea8/Sources/Amplitude/Public/AMPMiddleware.h#L57) change?

Reply:
no, this is how it is bridged to Swift

Reply:
Do you mind to point how this be called when the middleware has been removed? I didn't find `amplitudeDidRemove` in Amplitude-iOS .

Reply:
Swift will rename methods when they are bridged from ObjC. This is how it is renamed.

Reply:
Got it. Thanks.